modules/ROOT/pages/runtime-configure-eu.adoc

```diff
@@ -22,20 +22,21 @@ wrapper.java.additional.<n>=-Danypoint.platform.base_uri=https://eu1.anypoint.mu
 wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
 ----
 
-In the above example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
+In the previous example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
```

Comment:
See suggestion above.

If you keep the orig text, suggest using "In this example,..." because there is only one example in this subtopic.

Reply:
Updated this.